File: fizzbuzz/fizzbuzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> returnList = new ArrayList<>();
			for(int i =1; i <= n; i++) {
				if(i % 3 == 0) {
					
                    if(i % 5 == 0) {
                        returnList.add("FizzBuzz");
                        continue;
                    }
                
                    returnList.add("Fizz");
                    continue;
                }
				if(i % 5 ==0){
					returnList.add("Buzz");
                    continue;
				}
				
				returnList.add(Integer.toString(i));
                
				
			
		}
        return returnList;
    }
}

File: fizzbuzz/fizzbuzz.py
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        li = []

        for i in range(1,n+1,1):
            if(i % 3 == 0):
                if(i % 5 == 0):
                    li.append("FizzBuzz")
                    continue
                else:
                    li.append("Fizz")
                    continue
            
            if(i % 5 == 0):
                li.append("Buzz")
                continue
            
            
            li.append(str(i))
        return li
                
            